src/backend/services/file/service.py

```diff
@@ -1,95 +1,56 @@
-import os
-import re
```

Comment:
I realize I implemented this in the past, but let's remove the `file` folder, rename the `service.py` to `file.py` and move it in the `services` folder